bin/inspect_ocn.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# For manual/ocular inspection of a single OCN.
```

Comment:
I have a couple like this sitting in my local repo. We need to do some re-org at **some point** so we can both add all these helper type scripts and clean up some clutter.